### PR TITLE
[13.0][MIG] website_sale_tax_toggle: Migration

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT27484

@CarlosRoca13 @chienandalu Can you review?

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested :+1:

File: website_sale_tax_toggle/demo/assets.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<data>
+    <template id="assets_frontend_demo" inherit_id="website.assets_frontend">
```
Comment (CarlosRoca13) [inline website_sale_tax_toggle/demo/assets.xml]:
```suggestion
    <template id="assets_tests_demo" inherit_id="website.assets_tests">
```
On v13 exists the view assets_test. This assets are loaded just when testing, so you can place the file on templates and inherit from this view on data. Not need to place the assets on demo anymore

**Review by CarlosRoca13 — APPROVED** [review]
All working fine!! Just a non-blocking comment for tour loading

File: website_sale_tax_toggle/static/src/js/website_sale_tax_toggle.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* Copyright 2020 Sergio Teruel
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define("website_sale_tax_toggle.tax_toggle_button", function(require) {
+    "use strict";
+
+    var sAnimation = require("website.content.snippets.animation");
```
Comment (CarlosRoca13) [inline website_sale_tax_toggle/static/src/js/website_sale_tax_toggle.js]:
```suggestion
    const sAnimation = require("website.content.snippets.animation");
```

File: website_sale_tax_toggle/static/src/js/website_sale_tax_toggle.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    sAnimation.registry.tax_toggle_button = sAnimation.Class.extend({
+        selector: ".js_tax_toggle_management",
+        events: {
+            "click .js_tax_toggle_btn": "_onPublishBtnClick",
+        },
+        _onPublishBtnClick: function(ev) {
+            ev.preventDefault();
+            var $data = $(ev.currentTarget).parents(".js_tax_toggle_management:first");
```
Comment (CarlosRoca13) [inline website_sale_tax_toggle/static/src/js/website_sale_tax_toggle.js]:
```suggestion
            const $data = $(ev.currentTarget).parents(".js_tax_toggle_management:first");
```

File: website_sale_tax_toggle/static/src/js/website_sale_tax_toggle_tour.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+/* Copyright 2020 Sergio Teruel
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define("website_sale_tax_toggle.tour", function(require) {
+    "use strict";
+
+    var tour = require("web_tour.tour");
+    var base = require("web_editor.base");
```
Comment (CarlosRoca13) [inline website_sale_tax_toggle/static/src/js/website_sale_tax_toggle_tour.js]:
```suggestion
    const tour = require("web_tour.tour");
    const base = require("web_editor.base");
```

File: website_sale_tax_toggle/static/src/js/website_sale_tax_toggle_tour.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+    // Notice that it's important targeting the price as `span .oe_currency_value`
+    // or `.oe_price .oe_currency_value` to make sure this test is compatible
+    // with website_sale_b2x_alt_price module in this same repo.
+    var steps = [
```
Comment (CarlosRoca13) [inline website_sale_tax_toggle/static/src/js/website_sale_tax_toggle_tour.js]:
```suggestion
    const steps = [
```

**Review by CarlosRoca13 — COMMENTED** [review]
Don't saw in last review the js variable definition 😅

**Comment by sergio-teruel** [comment]
@CarlosRoca13 Changes done!!!

File: website_sale_tax_toggle/demo/assets.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<data>
+    <template id="assets_tests_demo" inherit_id="website.assets_tests_demo">
```
Comment (CarlosRoca13) [inline website_sale_tax_toggle/demo/assets.xml]:
```suggestion
    <template id="assets_tests_demo" inherit_id="website.assets_tests">
```

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
@CarlosRoca13 Sorry!! Changes done!!

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**